lib/rake/application.rb

```diff
@@ -94,10 +94,32 @@ def init(app_name="rake", argv = ARGV)
           # Backward compatibility for capistrano
           args = handle_options
         end
+        load_debug_at_stop
         collect_command_line_tasks(args)
       end
     end
 
+    def load_debug_at_stop_feature
+      return unless ['1', 'true'].include?(ENV["RAKE_DEBUG"])
```

Comment:
Should we this line simply like `return unless ENV["RAKE_DEBUG"]`?

Reply:
Yes, I agree with you.
This is a respecting the rspec-debug code. But simplifying it should be fine in most cases.